### PR TITLE
Fixing submission status to map properly to the front-end

### PR DESCRIPTION
**Pull request by nmonga91**

### CODE REVIEW

File: dataactcore/models/jobTrackerInterface.py [hunk 1]
```diff
@@ -243,22 +250,27 @@ def setNumberOfRowsById(self,jobId, numRows):
 
     def getSubmissionStatus(self,submissionId):
         jobIds = self.getJobsBySubmission(submissionId)
-        statuses = {"started": 0, "in_progress": 0, "has_errors": 0, "validated": 0}
+        statuses = {}
+        for status_name in self.getJobStatusNames():
+            statuses[status_name] = 0
+
```
Comment (jworcestBAH) [inline dataactcore/models/jobTrackerInterface.py]: This is the place for dict(zip(status_names, [0] * len(job_status_names))

File: dataactcore/models/jobTrackerInterface.py [hunk 2]
```diff
@@ -150,6 +150,13 @@ def markJobStatus(self,jobId,statusName):
         if prevStatus != 'finished' and statusName == 'finished':
             self.checkJobDependencies(jobId)
 
+    def getJobStatusNames(self):
+        """ Get All Job Status names """
+        job_status_names = []
+        for job_status in self.session.query(JobStatus.name):
+            job_status_names.append(str(job_status.name))
+        return job_status_names
+
```
Comment (jworcestBAH) [inline dataactcore/models/jobTrackerInterface.py]: So this would be 2 lines, self.getJobStatusId(None) and return JobStatus.JOB_STATUS_DICT.keys()

**Comment by klrBAH** [comment]
[run tests]

**Comment by nmonga91** [comment]
All comments have been addressed.

**Pull request merged**